go.mod

```diff
@@ -28,6 +28,7 @@ require (
 
 // versions are based on k8s.io/kubernetes
 replace (
+	github.com/intel/cri-resource-manager/pkg/topology v0.0.0 => ./pkg/topology
```

Comment:
Umm, what is this for?

Reply:
For being able to build cri-resmgr at all in CI. Without this go mod looks for it in a git repository at github.com/intel/cri-resource-manager/pkg/topology which then fails.